style.css

```diff
@@ -0,0 +1,525 @@
+  background-color: #fdb431;
+}
+
+.catalog-item__brown {
+  height: 7px;
+  width: 7px;
+  border-radius: 50%;
+  background-color: #e29d69;
+}
+
+.catalog-item__cherry {
+  height: 7px;
+  width: 7px;
+  border-radius: 50%;
+  background-color: #762e31;
+}
+
+.catalog-item__blue {
+  height: 7px;
+  width: 7px;
+  border-radius: 50%;
+  background-color: #257da9;
+}
+
+.active-item {
+  height: 9px;
+  width: 9px;
+}
+
+.gradient {
+  display: none;
+}
+
+.gradient-reverse {
+  display: none;
+}
+
+/* video-section */
+
+.video-section {
+  width: 100%;
+  height: 442px;
+  background-color: #000;
+  display: flex;
+  align-items: center;
+  margin-bottom: 95px;
+  justify-content: space-around;
+}
+
+.video-section__image {
+  display: block;
+}
+
+.video-adaptive {
+  display: none;
+}
+
+.video-section__content {
+  display: block;
+}
+
+.video-content__h3 {
+  width: 258px;
+  font-size: 22px;
+  line-height: 27px;
+  margin-bottom: 12px;
+  color: #fff;
+  text-transform: uppercase;
+}
+
+.video-content__description {
+  width: 248px;
+  font-size: 13px;
+  line-height: 16px;
+  margin-bottom: 10px;
+  color: #fff;
+}
+
+/* stories-content */
+
+.stories-content {
+  width: 100%;
+  padding-bottom: 90px;
+}
+
+.stories-content__h4 {
+  font-size: 22px;
+  font-weight: bold;
+  line-height: 21px;
+  text-align: center;
+  text-transform: uppercase;
+  margin-bottom: 40px;
+}
+
+.stories-items {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+}
+
+.stories-items__item {
+  width: 277px;
+}
+
+.stories-item__image {
+  display: block;
+  width: 100%;
+  margin-bottom: 30px;
+}
+
+.stories-item__title {
+  font-size: 18px;
+  line-height: 17px;
+  font-weight: bold;
+  margin-bottom: 8px;
+}
+
+.stories-item__description {
+  font-size: 13px;
+  line-height: 16px;
+  margin-bottom: 18px;
+}
+
+.stories-item__description-extended {
+  width: 100%;
+  display: grid;
+  grid-template-columns: 0.9fr 1fr 1fr;
+}
+
+.stories-item__date {
+  font-size: 12px;
+  line-height: 11px;
+  font-weight: bold;
+  color: #000;
+  opacity: 0.3;
+  text-align: right;
+}
+
+/* gallery */
+
+.gallery {
+  width: 100%;
+  padding-bottom: 94px;
+}
+
+.gallery__title {
+  font-size: 18px;
+  line-height: 17px;
+  color: #000;
+  opacity: 0.3;
+  text-align: right;
+  text-transform: uppercase;
+  font-weight: bold;
+  margin-bottom: 8px;
+}
+
+.gallery-content {
+  display: grid;
+  grid-template-columns: repeat(7, 1fr);
+}
+
+.gallery-content__item {
+  display: block;
+}
+
+.gallery__photo {
+  width: 100%;
+}
+
+.hidden-1024 {
+  display: none;
+}
+
+.footer {
+  background-color: #000;
+  background-size: cover;
+  font-family: 'GothamPro', serif;
+  margin: 0 auto;
+  box-sizing: border-box;
+}
+
+.footer__content {
+	max-width: 930px;
+  margin: 0 auto;
+  padding-top: 69px;
+  display: flex;
+  justify-content: space-around;
+}
+
+.footer__column {
+  display: block;
+}
+
+.footer__column-title {
+	margin-bottom: 10px;
+	color: #fff;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/PolinaYartseva_axion/pullRequest?prid=5569682)